luigi/databases/ensembl/helpers/ensembl.py

```diff
@@ -102,8 +102,8 @@ def locus_description(entry):
     return '{species} {rna_type} {locus_tag}'.format(
         species=species,
         rna_type=entry.rna_type.replace('_', ' '),
-        locus_tag=entry.locus_tag,
-    )
+        locus_tag=entry.locus_tag or '',
```

Comment:
Just wondering if `or ''` needs to be added anywhere else in similar cases?

Reply:
I don't think so, the other parts that will always show up are rna_type and species, both of which must exist.